Application/EdFi.Ods.Common/Configuration/ApiSettings.cs

```diff
@@ -50,6 +53,27 @@ public ApiSettings()
 
         public bool? UseReverseProxyHeaders { get; set; }
 
+        public string DefaultForwardingHostServer
+        {
+            get
+            {
+                return _defaultForwardingHostServer ?? "localhost";
+            }
+            set
+            {
+                _defaultForwardingHostServer = value;
+            }
+        }
+
+        public int DefaultForwardingHostPort
```

Comment:
There is an error if the api is started with `DefaultForwardingHostPort = ""` (and `UseReverseProxyHeaders = false`), since this property is an _int_ it always expects a valid number during startup.

I recommend changing it to a nullable _int_ (`int?`) and instead of relying on fallback values (like _localhost_ and port _80_ here, or port _443_ on _HttpRequestExtensions_ methods parameters) let them default to null, and if ``UseReverseProxyHeaders = true`` and any of the related _DefaultForwardingHostPort_ or _DefaultForwardingHostServer_ is invalid throw an error instructing the user to check his appsettings.

Thoughts?